pkg/features/antrea_features.go

```diff
@@ -177,6 +178,50 @@ var (
 		AdminNetworkPolicy:          {Default: false, PreRelease: featuregate.Alpha},
 	}
 
+	// AgentGates consists of all known feature gates for the Antrea agent.
+	// When adding a new feature gate that applies to the Antrea agent, please also add it here.
```

Comment:
```suggestion
	// AgentGates consists of all known feature gates for the Antrea Agent.
	// When adding a new feature gate that applies to the Antrea Agent, please also add it here.
```
Antrea Agent and Antrea Controller are preferred in a comment or a log.